src/utils/functionGeneration.ts

```diff
@@ -48,6 +48,7 @@ export function createCairoFunctionStub(
   implicits: Implicits[],
   ast: AST,
   nodeInSourceUnit: ASTNode,
+  splitDarray?: boolean,
```

Comment:
Not sure how I feel about this being an argument name, I didn't want to add anything like "externalInterface" since the functions should always be internal.